Sift.sln

```diff
@@ -1,11 +1,15 @@
 Microsoft Visual Studio Solution File, Format Version 12.00
 # Visual Studio Version 17
-VisualStudioVersion = 17.5.33414.496
+VisualStudioVersion = 17.6.33723.286

```

Comment:
does this affect the customer's integration with `sift-dotnet` somehow?

Reply:
No it will not